.github/workflows/test.yml

```diff
@@ -50,6 +50,17 @@ jobs:
         with:
           flags: basic
           functionalities: gcov
+  validate-legacy-node:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 10.x
+          cache: "yarn"
+      - run: node -e "const content = require('./package.json');delete content.devDependencies;require('fs').writeFileSync('package.json', JSON.stringify(content, null, 2));"
+      - run: yarn install --production --frozen-lockfile
```

Comment:
This line validate that we can install our deps on Node.js v10 (and so above), so if somebody tried to update our deps and it will not compatibility with node.js 10 we it will be failed